server/src/router/uploadRouter.ts

```diff
@@ -62,6 +64,7 @@ uploadRouter.post(
 
 uploadRouter.patch(
   "/:id",
+  isPlanner,
```

Comment:
Yes, but this is only authorization middleware. Every route with protected access should have also the authentication middleware `isLoggedIn`